7.4/fpm/ubuntu/build/install_xdebug.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -x
+
+apt-get update && apt-get install php-pear
+
+pecl install xdebug-beta && docker-php-ext-enable xdebug
```

Comment:
Did some fiddling and found that you don't have to use the `docker-php-ext-enable` script.

Before `pecl install` you can update config entries for `pecl` and `pear`. This way `pecl` will automatically enable the xdebug extension.

```
pecl config-set php_init $PHP_INI_DIR/php.ini
pear config-set php_init $PHP_INI_DIR/php.ini
```

Having this, I think that you can entirely remove the `docker-php-ext-enable` script.

Reply:
Ah, nice.